modules/theme/src/themes/default/modules/modal.overrides

```diff
@@ -0,0 +1,200 @@
+/*!
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+.ui.modal {
+    &.custom-modal {
+        text-align: @customModalContentAlignment;
+
+        .modal-icon {
+            margin-top: @customModalIconMarginTop;
+        }
+        .modal-description {
+            margin: @customModalDescriptionMarginTop;
+        }
+        .modal-heading {
+            margin: @customModalHeadingMarginTop;
+        }
+    }
+}
+
+.svg-box {
+    display:inline-block;
+    position: relative;
+    width:150px;
+    margin-bottom: -25px !important;
+}
+
+.green-stroke {
+    stroke:#7CB342;
+}
+
+.red-stroke {
+    stroke: #FF6245;
+}
+
+.yellow-stroke {
+    stroke: #FFC107;
+}
```

Comment:
Shall we scope these styles with `svg-box`? And shall we use the `negative, warning` ... etc terms?